cartography/intel/aws/wafv2.py

```diff
@@ -0,0 +1,191 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import botocore
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.intel.aws.ec2.util import get_botocore_config
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_load_waf_resources(client: botocore.client.BaseClient,
+                           elbv2_client: botocore.client.BaseClient, acl_arn: str) -> Dict[str, List[str]]:
+    resources: Dict[str, List[str]] = {'ALBs': [], 'APIGWs': []}
+    resources["APIGWs"].extend(
+        client.list_resources_for_web_acl(WebACLArn=acl_arn, ResourceType='API_GATEWAY')['ResourceArns'])
```

Comment:
Does this also need to be paginated?

Try to use our `aws_paginate()` un util.py
https://github.com/lyft/cartography/blob/3ae3ad1f9740f3f69894f2d0514125f908bd5f8d/cartography/util.py#L138

Reply:
this resource doesn't support pagination